tabs/configuration.js

```diff
@@ -190,6 +190,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             escprotocols.push('BRUSHED');
         }
 
+        if (CONFIG.flightControllerIdentifier == "BTFL" && semver.gte(CONFIG.flightControllerVersion, "3.1.0")) {
```

Comment:
Can you please remove `CONFIG.flightControllerIdentifier == "BTFL"`? This is not needed any more.